plugins/cost-insights/src/components/CostOverviewCard/CostOverviewCard.tsx

```diff
@@ -49,6 +49,15 @@ export const CostOverviewCard = ({
   const config = useConfig();
   const [tabIndex, setTabIndex] = useState(0);
 
+  // Reset tabIndex if breakdowns available change
+  useEffect(() => {
+    // Intentionally off-by-one to account for the overview tab
+    const lastIndex = Object.keys(dailyCostData.groupedCosts ?? {}).length;
+    if (tabIndex > lastIndex) {
```

Comment:
ooc, when do we expect this to be the case ?

Reply:
A realistic example is having breakdowns by product + project for daily cost, then adding a project filter. The "breakdown by project" no longer makes any sense and may not be included.

Reply:
@timbonicus ah, makes sense thanks